_includes/css/base/typography.css

```diff
@@ -158,7 +156,7 @@ ol li:last-child {
 /* Other elements */
 
 a {
-  text-decoration-color: rgba(var(--c-link-text-rgb, var(--c-accent-dark-rgb)), 0.6);
+  text-decoration-color: rgb(var(--c-link-text-rgb, var(--c-accent-dark-rgb)) 0.6);
```

Comment:
@LynnHerrick9 I think you might have omitted a slash before the `0.6` here. (Ignore me if this is intentional!)

Reply:
This was the recommended change from the linter but I recognize that it doesn't match the [documented example](https://stylelint.io/user-guide/rules/color-function-notation/).  To check this I have inspected the page, scrolled down to the links at the bottom and manually unselected `.footer a` styles so it falls back to `a` and the `text-decoration-color` gets picked up on these links.  There appears to be no change between running both notations but  I can add the slash since this leans towards what I am seeing in the documentation.